cumulus_library/template_sql/codeable_concept_denormalize.sql.jinja

```diff
@@ -120,13 +120,12 @@ CREATE TABLE {{ target_table }} AS (
             ROW_NUMBER()
                 OVER (
                     PARTITION BY id
-                    ORDER BY priority ASC
+                    ORDER BY priority ASC, code ASC
```

Comment:
For better predictability in tests (shouldn't normally happen that you have multiple codes for the same system)